sandbox/replication.go

```diff
@@ -313,8 +315,9 @@ func CreateMasterSlaveReplication(sandboxDef SandboxDef, origin string, nodes in
 			return err
 		}
 	}
-	fmt.Printf("Replication directory installed in %s\n", common.ReplaceLiteralHome(sandboxDef.SandboxDir))
-	fmt.Printf("run 'dbdeployer usage multiple' for basic instructions'\n")
+	// TODO: Improve logging
```

Comment:
I will do for replication the same thing that I do for single sandbox. When running in `dbdeployer mode`, it shows the help messages.